packages/augur-ui/src/modules/markets/reducers/new-market.ts

```diff
@@ -7,73 +7,17 @@ import {
 } from 'modules/markets/actions/update-new-market';
 import { RESET_STATE } from 'modules/app/actions/reset-state';
 import {
-  SETTLEMENT_FEE_DEFAULT,
-  EXPIRY_SOURCE_GENERIC,
-  DESIGNATED_REPORTER_SELF,
-  AFFILIATE_FEE_DEFAULT,
-  YES_NO,
-  YES_NO_OUTCOMES,
   ZERO,
-  ONE,
   NEW_ORDER_GAS_ESTIMATE,
 } from 'modules/common/constants';
 import { createBigNumber } from 'utils/create-big-number';
 import { NewMarket, BaseAction, LiquidityOrder } from 'modules/types';
 import { formatShares, formatDai } from 'utils/format-number';
+import { EMPTY_STATE } from 'modules/create-market/constants';
 
-export const DEFAULT_STATE: NewMarket = {
-  isValid: false,
-  validations: {
-    description: null,
-    categories: ['', '', ''],
-    designatedReporterAddress: null,
-    expirySourceType: null,
-    setEndTime: null,
-    hour: null,
-    minute: null,
-    meridiem: null,
-    scalarDenomination: null,
-    outcomes: ['', ''],
-    settlementFee: '',
-  },
-  currentStep: 0,
-  marketType: YES_NO,
-  outcomes: ['', ''],
-  outcomesFormatted: YES_NO_OUTCOMES,
-  scalarSmallNum: '',
-  scalarBigNum: '',
-  scalarDenomination: '',
-  description: '',
-  expirySourceType: EXPIRY_SOURCE_GENERIC,
-  expirySource: '',
-  backupSource: '',
-  designatedReporterType: DESIGNATED_REPORTER_SELF,
-  designatedReporterAddress: '',
-  endTime: null,
-  setEndTime: null,
-  tickSize: 0.01,
-  hour: null,
-  minute: null,
-  meridiem: 'AM',
-  offset: 0,
-  offsetName: null,
-  timezone: null,
-  detailsText: '',
-  categories: ['', '', ''],
-  settlementFee: SETTLEMENT_FEE_DEFAULT,
-  affiliateFee: AFFILIATE_FEE_DEFAULT,
-  orderBook: {}, // for submit orders
-  orderBookSorted: {}, // for order book table
-  minPrice: '0',
-  maxPrice: '1',
-  minPriceBigNumber: ZERO,
-  maxPriceBigNumber: ONE,
-  initialLiquidityDai: ZERO,
-  initialLiquidityGas: ZERO,
-};
-
+const bob = JSON.parse(JSON.stringify(EMPTY_STATE));
```

Comment:
did you mean to keep this

Reply:
yes, only way I could make sure default state was actually default state.